js/src/figure.js

```diff
@@ -580,10 +557,17 @@ var FigureView = widgets.DOMWidgetView.extend( {
             return make_line(-0.5, -0.5, -0.5 ,  -0.5+x, -0.5+y, -0.5+z, material)
         }
         var linewidth = 1;
-        this.axes_material = new THREE.LineBasicMaterial({color: "cyan", linewidth: linewidth});
-        this.xaxes_material = new THREE.LineBasicMaterial({color: "red", linewidth: linewidth});
-        this.yaxes_material = new THREE.LineBasicMaterial({color: "green", linewidth: linewidth});
-        this.zaxes_material = new THREE.LineBasicMaterial({color: "blue", linewidth: linewidth});
+        var material_parameters = {
+            linewidth: linewidth,
+            transparent: true,
+            depthWrite: true, 
+            depthTest: true,
```

Comment:
There are already the default right https://threejs.org/docs/#api/materials/Material ?

Reply:
Yes, but I had them there during debugging some depth problems.